mysql-test/suite/innodb/t/percona_changed_page_bmp_1368530.test

```diff
@@ -0,0 +1,34 @@
+#
+# Test that innodb_track_changed_pages couldn't be enabled in runtime (bug 1368530)
+
+--source include/have_innodb.inc
+--source include/have_debug.inc
+
+--Error ER_WRONG_VALUE_FOR_VAR
+SET @@GLOBAL.innodb_track_changed_pages=TRUE;
+# purge causes crash described in lp:1368530
+PURGE CHANGED_PAGE_BITMAPS BEFORE 0;
+
+--exec echo "wait" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--shutdown_server 10
+--source include/wait_until_disconnected.inc
+
+--enable_reconnect
+--echo 1st restart
+--exec echo "restart: --innodb_track_changed_pages=1" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--source include/wait_until_connected_again.inc
+
+SET @@GLOBAL.innodb_track_changed_pages=FALSE;
+SELECT @@GLOBAL.innodb_track_changed_pages;
+
+SET @@GLOBAL.innodb_track_changed_pages=TRUE;
+SELECT @@GLOBAL.innodb_track_changed_pages;
+
+--exec echo "wait" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--shutdown_server 10
+--source include/wait_until_disconnected.inc
+
+--enable_reconnect
+--echo 2nd restart
+--exec echo "restart" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--source include/wait_until_connected_again.inc
```

Comment:
If you removed tmpdir / rmdir, then we need to do RESET BITMAPS in order not to leave stray xdb around